### ISSUE TITLE
missing ipxe binary artifact in on-imagebuilder

### ISSUE THREAD

**Issue by heckj** [RackHD/RackHD#345]
from @codenrhoden 

> Any idea where I can find the file `monorail.intel.ipxe`, as referenced here: > https://github.com/RackHD/on-dhcp-proxy/blob/master/lib/message-handler.js#L257 ? 
> I tried grabbing it from bintray, as the `files` Docker image grabs the rest from a URL like this: "https://bintray.com/artifact/download/rackhd/binary/ipxe/monorail.ipxe" but “monorail.intel.ipxe” does not exist there

**Comment by heckj** [comment]
log into an ubuntu box (or spin up a vagrant instance):
 - `git clone https://github.com/rackhd/on-imagebuilder`
 - `cd on-imagebuilder`
 - run the iPXE build generation process
 - `sudo ansible-playbook -i hosts common/ipxe_builder.yml -e "config_file=vars/ipxe.yml"`

This builds iPXE in `/tmp/on-imagebuilder/ipxe_build_env/ipxe/src`

- `cd /tmp/on-imagebuilder/ipxe_build_env/ipxe/src`
- `sudo make bin/intel.pxe EMBED=default.ipxe`

grab the file at `/tmp/on-imagebuilder/ipxe_build_env/ipxe/src/bin/intel.pxe` and rename it to "monorail.intel.ipxe" and put it into the same TFTP static files location with the other bootloaders on your system (if you're doing this from source, that's `~/src/on-tftp/static/tftp`, if from debian packages, it's more likely `/var/renasar/on-tftp/static/tftp`.

**Comment by codenrhoden** [comment]
The ipxe image this generates works fine for my NUC.  Thanks!  Once the fix is merged into on-imagebuilder, does the file get automically pushed up to bintray?

**Comment by heckj** [comment]
@codenrhoden yep, it's dropped into the right place so it'll automatically populate. I've just put up another PR that will update our vagrant demo instance to make sure it gets added to local VMs as well when someone uses Ansible to install: https://github.com/RackHD/RackHD/pull/349
